ambari-web/app/utils/validator.js

```diff
@@ -264,7 +264,7 @@ module.exports = {
       return true;
     };
     if (/^[\?\|\*\!,]/.test(value)) return false;
-    return /^((\.\*?)?([\w\s\[\]\/\?\-_,\|\*\!\{\}\(\)]*)?)+(\.\*?)?$/g.test(value) && (checkPair(['[',']'])) && (checkPair(['{','}']));
+    return /^((\.\*?)?([\w\s\[\]\/\?\-_,\|\*\!\{\}\(\)]|[\u00A0-\uD7FF\uF900-\uFDCF\uFDF0-\uFFEF]]*)?)+(\.\*?)?$/g.test(value) && (checkPair(['[',']'])) && (checkPair(['{','}']));
```

Comment:
The diapasons here are related to:
- Latin block (characters from dead language and used in medicine, non English characters)
- CJK with Latin (wrong and not fully covered)
- variation of symbols (non characters) and some Arabic range

Please referrer here for correct CJK ranges: 
 https://stackoverflow.com/questions/1366068/whats-the-complete-range-for-chinese-characters-in-unicode

and you might include Japanese ranges as well

To verify what is in the range, could be used (found with google) `https://qaz.wtf/u/show.cgi`. Mode `Hexadecimal range`, value for example : `\u00A0-\uD7FF` ->  `00A0-D7FF`